shared/test/unit_test/os_interface/linux/xe/ioctl_helper_xe_tests.cpp

```diff
@@ -835,6 +835,61 @@ TEST(IoctlHelperXeTest, givenComputeDssWhenGetTopologyDataAndMapThenResultsAreCo
     }
 }
 
+TEST(IoctlHelperXeTest, givenSimd16EuPerDssAndEuPerDssWhenGetTopologyDataAndMapThenPreferSimd16EuPerDss) {
```

Comment:
it's not meant to prefer one vs the other as we don't return both for any platform.  For a non-mock test, an assert that this is true would be ok.